samples/cpp/benchmark_app/main.cpp

```diff
@@ -186,6 +190,9 @@ void setDeviceProperty(ov::Core& core,
 int main(int argc, char* argv[]) {
     std::shared_ptr<StatisticsReport> statistics;
     try {
+#ifdef __WINDOWS__
+        DisableMinimizeButton(GetForegroundWindow());
```

Comment:
I don't think that we should disable minimize button from benchmark_app.  If we have potential problems with performance on windows with background process, it should be described in documentation with appriopriate recommendations. Looks like workaround where becnhmarking is doing just for becnhmarking

Reply:
I accidentally rebased this from my other PR, but thank you for your input - I removed these lines and will update the documentation